### ISSUE TITLE
Bug in torch BERT classifier. Wrong working with model outputs.

### ISSUE THREAD

**Issue by castargo** [deepmipt/DeepPavlov#1405]
https://github.com/deepmipt/DeepPavlov/blob/f9af2fa3f9f425e715aae647b57637406fc7145e/deeppavlov/models/torch_bert/torch_transformers_classifier.py#L120

This model returns dict like object that leads to invalid values in variables - loss and logits are just strings "loss" and "logits".

**Comment by yurakuratov** [comment]
Hi! 

What version of `transformers` is installed?
DeepPavlov is currently using `transformers==2.8.0`:
https://github.com/deepmipt/DeepPavlov/blob/master/deeppavlov/requirements/transformers.txt#L1

**Comment by castargo** [comment]
Sorry, I used the last stable version of transformers `transformers==4.3.3`. With 2.8.0 all work fine. 
I hope my [PR](https://github.com/deepmipt/DeepPavlov/pull/1406) can be useful if you will migrate to transformers 4 in the future. 

Thank you!

**Comment by yurakuratov** [comment]
> I hope my PR can be useful if you will migrate to transformers 4 in the future.

Sure, thank you.
